src/main/java/org/broadinstitute/hellbender/utils/GenotypeUtils.java

```diff
@@ -136,8 +136,14 @@ public static GenotypeCounts computeDiploidGenotypeCounts(final VariantContext v
         return new GenotypeCounts(genotypeWithTwoRefsCount, genotypesWithOneRefCount, genotypesWithNoRefsCount);
     }
 
+    /**
+     * Do we have (or can we infer) likelihoods necessary for allele frequency calculation?
+     * Some reblocked and/or DRAGEN GVCFs omit likelihoods for ref blocks, but we can estimate them
```

Comment:
Is there a test case for this method that uses such reblocked / DRAGEN data as input?

Reply:
testWithReblockedGVCF tests with hom-ref blocks that have no PLs, like:
`20      10087821        .       A       <NON_REF>       .       .       END=10087822    GT:DP:GQ        0/0:93:60       ./.
20      10433000        .       G       <NON_REF>       .       .       END=10433297    GT:DP:GQ        0/0:36:60       0/0:38:60
20      10433298        .       G       <NON_REF>       .       .       END=10433301    GT:DP:GQ        0/0:35:50       0/0:38:50`